quipucords/scanner/network/connect_callback.py

```diff
@@ -26,12 +30,15 @@ class ConnectResultCallback():
     scan, as we scan it.
     """
 
-    # pylint: disable=protected-access
-    def __init__(self, result_store, credential, source):
+    # pylint: disable=protected-access,too-many-arguments
+    def __init__(self, result_store, credential, source,
+                 manager_interrupt):
         """Create result callback."""
         self.result_store = result_store
         self.credential = credential
         self.source = source
+        self.interrupt = manager_interrupt
+        self.stop = False
```

Comment:
Let's call this self.stopped since it indicates that it has been stopped verse a command to stop.